From c98f84fea6a11fcea7410a6e700da39f94421cd2 Mon Sep 17 00:00:00 2001
From: Mike DePaulo <mikedep333@gmail.com>
Date: Fri, 3 Apr 2015 08:37:03 -0400
Subject: Change SPICE passwords limited to 60 characters

The limit for SPICE passowrds is much more than 8 characters.

https://bugzilla.gnome.org/show_bug.cgi?id=747296
---
 plugins/spice/vinagre-spice-tab.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/plugins/spice/vinagre-spice-tab.c b/plugins/spice/vinagre-spice-tab.c
index 0a26a00..38fb53b 100644
--- a/plugins/spice/vinagre-spice-tab.c
+++ b/plugins/spice/vinagre-spice-tab.c
@@ -36,6 +36,9 @@
 
 #define VINAGRE_SPICE_TAB_GET_PRIVATE(object)(G_TYPE_INSTANCE_GET_PRIVATE ((object), VINAGRE_TYPE_SPICE_TAB, VinagreSpiceTabPrivate))
 
+/* From spice-protocol.h */
+#define SPICE_MAX_PASSWORD_LENGTH 60
+
 typedef struct _VinagreSpiceDisplay
 {
   GtkWidget  *display;
@@ -299,7 +302,7 @@ spice_main_channel_event_cb(SpiceChannel *channel, SpiceChannelEvent event,
 				       name,
 				       FALSE,
 				       TRUE,
-				       8,
+				       SPICE_MAX_PASSWORD_LENGTH,
 				       NULL,
 				       &password,
 				       &save_in_keyring))
-- 
cgit v0.12

